src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -379,13 +379,13 @@ double BaseCouplingScheme::getNextTimestepMaxLength() const
 bool BaseCouplingScheme::isCouplingOngoing() const
 {
   bool timeLeft      = math::greater(_maxTime, _time, _eps) || math::equals(_maxTime, UNDEFINED_TIME);
-  bool timestepsLeft = (_maxTimesteps >= _timesteps) || (_maxTimesteps == UNDEFINED_TIMESTEPS);
+  bool timestepsLeft = (_maxTimesteps >= _timesteps) || (_maxTimesteps == UNDEFINED_TIME_WINDOWS);
```

Comment:
`timestepsLeft`, `_maxTimesteps` and `_timesteps` should be renamed, as well.

Reply:
That's too far down the rabbit hole. I did not update variables in `cplscheme` outside the configuration. I opened #639 for that.